HerPublicWebsite/Program.cs

```diff
@@ -33,9 +34,17 @@ public static void Main(string[] args)
             app
                 .Services
                 .GetService<IRecurringJobManager>()
-                .AddOrUpdate<RegularJobsService>(
-                    "Nightly tasks",
-                    rjs => rjs.RunNightlyTasksAsync(),
+                .AddOrUpdate<ReferralFollowUpService>(
+                    "Get referrals passed ten day working threshold with no follow up",
+                    rjs => rjs.GetReferralsPassedTenWorkingDayThresholdWithNoFollowUp(),
+                    "30 0 * * *");

```

Comment:
If there's no particular requirement for this to be 00.30, it might make sense to separate it slightly from the other regular job (e.g. 00:45) - that way the logs will be neatly separated

Reply:
Good point. I'm not aware of any reason for them to occur at the same time. I'll update this.